Makefile

```diff
@@ -4,20 +4,15 @@ FEDORA_VERSION ?= f32
 default:  help
 
 LIB_T  := $(PREFIX)/var/lib/systemd-swap
-ETCD_T := $(PREFIX)/etc/systemd/swap.conf.d/
-
 BIN_T  := $(PREFIX)/usr/bin/systemd-swap
 SVC_T  := $(PREFIX)/lib/systemd/system/systemd-swap.service
-CNF_T  := $(PREFIX)/etc/systemd/swap.conf
+CNF_T  := $(PREFIX)/usr/share/systemd-swap/swap.conf
```

Comment:
We need something in `/etc/systemd/swap.conf` before the new release, but this will do for now

Reply:
I am preparing another commit for this.

Reply:
This should be good to go. The name choice could definitely be improved, but it should suffice for now since the exact location of the default configuration should be irrelevant to users anyway.